salt/states/file.py

```diff
@@ -929,10 +929,10 @@ def symlink(
                     return _error(ret, ((
                                             'File exists where the backup target {0} should go'
                                         ).format(backupname)))
-                elif os.path.isfile(backupname):
-                    os.remove(backupname)
-                elif os.path.isdir(backupname):
-                    shutil.rmtree(backupname)
+                elif os.path.isfile(backupname)\
```

Comment:
I don't understand why that would be relevent here, since it seems like the intention is to delete whatever is passed regardless of whether or not it is a symlink. At least, that's how I read the `force` option.